microcosm_flask/decorators/schemas.py

```diff
@@ -0,0 +1,57 @@
+from copy import deepcopy
+
+from inflection import camelize, underscore
+from marshmallow import Schema
+
+from microcosm_flask.naming import name_for
+from microcosm_flask.swagger.naming import type_name
+
+
+def _get_fields_and_make_required(schema_cls, selected_fields):
+    associated_fields = {}
+    for field_name in selected_fields:
+        field = deepcopy(schema_cls._declared_fields[field_name])
+        field.required = True
```

Comment:
We don't want to be able to specify non-required fields? I know we may not have a use case, but others might.

Reply:
I hesitated, because making all fields required just simplifies the API. I guess I could leave the option specify optional field, and make a shorthand version that makes all fields required.